### PR TITLE
searchbar: enable suggestions on searchbar component

### PR DESCRIPTION
**Pull request by zzacharo**

### CODE REVIEW

File: examples/simple/index.html [hunk 1]
```diff
@@ -22,14 +22,21 @@
 <html>
   <head>
     <script src="./node_modules/angular/angular.js"></script>
+    <script src="./node_modules/angular-mass-autocomplete/massautocomplete.js"></script>
+    <script src="./node_modules/angular-sanitize/angular-sanitize.js"></script>
     <script src="./node_modules/d3/d3.js"></script>
     <script src="./node_modules/d3-tip/index.js"></script>
+    <script src="node_modules/lodash/lodash.js"></script>
```
Comment (drjova) [inline examples/simple/index.html]: AFAIR we don't use ``lodash`` in this repo ☺️

File: examples/simple/index.html [hunk 2]
```diff
@@ -22,14 +22,21 @@
...
     <script src="/dist/invenio-search-js.js"></script>
     <link href="./node_modules/bootstrap/dist/css/bootstrap.css" rel="stylesheet"/>
+    <link href="./node_modules/angular-mass-autocomplete/massautocomplete.theme.css"     rel="stylesheet"/>
```
Comment (drjova) [inline examples/simple/index.html]: space :)

File: examples/simple/index.html [hunk 3]
```diff
@@ -40,7 +47,7 @@
             <h1>Invenio Search Demo</h1>
           </div>
           <div ng-app="invenioSearch">
-            <invenio-search search-endpoint="http://localhost:5000/api/records/">
+            <invenio-search search-endpoint="https://cdslabs-qa.cern.ch/api/records/">
```
Comment (drjova) [inline examples/simple/index.html]: oups

File: src/invenio-search-js/templates/results.html [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <ul>
   <li ng-repeat="record in vm.invenioSearchResults.hits.hits track by $index">
-    <h5>{{ record.metadata.control_number }}: {{ record.metadata.title_statement.title }}</h5>
-    <p>{{ record.metadata.summary[0].summary }}</p>
+    <h5>{{ record.metadata.report_number[0] }}: {{ record.metadata.title.title }}</h5>
```
Comment (drjova) [inline src/invenio-search-js/templates/results.html]: hmm the default template should be compatible with invenio default data model

**Comment by zzacharo** [comment]
Closing as this is not needed anymore

**Pull request closed without merging**